compiler/ir/ir.psi2ir/src/org/jetbrains/kotlin/psi2ir/intermediate/ExtensionInvokeCallReceiver.kt

```diff
@@ -34,6 +34,6 @@ class ExtensionInvokeCallReceiver(
             "Extension 'invoke' call should have null as its 1st value argument, got: ${callBuilder.irValueArgumentsByIndex[0]}"
         }
         callBuilder.irValueArgumentsByIndex[0] = extensionInvokeReceiver.load()
-        return withDispatchAndExtensionReceivers(functionReceiver, null)
+        return withDispatchAndExtensionAndContextReceivers(functionReceiver, null, emptyList())
```

Comment:
The following code is correct:
```
fun of42(fn: Int.() -> Int) = 42.fn()

fun test(fn: (Int) -> Int) = of42(fn)
```
Please, add a corresponding test using a function-with-context-receiverts type if that's possible.

Reply:
Added in [this commit](https://github.com/JetBrains/kotlin/pull/4488/commits/cdf1e5283ebf50b2278c79bec7ebf397e5d02d21).